src/templates/ts/application/package.json

```diff
@@ -1,61 +1,60 @@
 {
-  "name": "<%= name %>",
-  "version": "<%= version %>",
-  "description": "<%= description %>",
-  "author": "<%= author %>",
+  "name": "nest-typescript-starter",
+  "version": "1.0.0",
+  "description": "Nest TypeScript starter repository",
   "license": "MIT",
   "scripts": {
-    "format": "prettier --write \"**/*.ts\"",
+    "format": "prettier --write \"src/**/*.ts\"",
     "start": "ts-node -r tsconfig-paths/register src/main.ts",
     "start:dev": "nodemon",
-    "prestart:prod": "rm -rf dist && tsc",
+    "start:debug": "nodemon --config nodemon-debug.json",
+    "prestart:prod": "rimraf dist && tsc",
     "start:prod": "node dist/main.js",
     "start:hmr": "node dist/server",
+    "lint": "tslint -p tsconfig.json -c tslint.json",
     "test": "jest",
+    "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:e2e": "jest --config ./test/jest-e2e.json",
     "webpack": "webpack --config webpack.config.js"
   },
   "dependencies": {
-    "@nestjs/common": "^5.0.0",
-    "@nestjs/core": "^5.0.0", <% if (dependencies.indexOf('microservices') !== -1) { %> 
-    "@nestjs/microservices": "^5.0.0", <% } %><% if (dependencies.indexOf('websockets') !== -1) { %> 
-    "@nestjs/websockets": "^5.0.0", <% } %>
```

Comment:
not sure if we should remove it^

Reply:
if we leave it we can start common and core only.
I think we don't need microservices and websockets for an frontend rest api that calls services and aggregate data for a frontend web client, don't you ?

Reply:
same as below, not sure why it's removed